app/components/UserSelect/index.js

```diff
@@ -29,7 +29,10 @@ const UserSelect = ({
             : [value.id]
           : []),
       ],
-      availabilityDuring: availabilityStatusDuring,
+      availabilityDuring: availabilityStatusDuring || {
+        start_at: 0,
+        end_at: 0,
+      },
```

Comment:
**Hep here:** required to make changes here,  if want to hide the status in Impersonate dropdown then availabilityStatusDuring must be undefined and without {    start_at: 0,    end_at: 0, }, dropdown options were not coming.